src/convertToObject.js

```diff
@@ -13,6 +13,26 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const arrayOfValues = sourceString.split(';');
+  const object = arrayOfValues.reduce((prev, element) => {
+    const arrTemp = element.split(':');
```

Comment:
```suggestion
    const [key, value] = element.split(':');
```
Нужные значения сразу доставай из массива, а в if проверяй key.trim() и value.trim()